### PR TITLE
Add missing boolean interior parameter to ClrArray.GetStructValue() 

### PR DESCRIPTION
**Pull request by chrisnas**
This is needed to support array of struct: I'm not sure that false is the right default value but it was the initial implementation.

### CODE REVIEW

File: src/Microsoft.Diagnostics.Runtime/src/Common/ClrArray.cs [hunk 1]
```diff
@@ -199,7 +199,7 @@ private ulong GetItemAddress(int index, bool isObjectReference)
             }
         }
 
-        public ClrValueType GetStructValue(int index)
+        public ClrValueType GetStructValue(int index, bool interior = false)
```
Comment (leculver) [inline src/Microsoft.Diagnostics.Runtime/src/Common/ClrArray.cs:202]: This is technically a breaking change.  I think we'll need to structure it this way to preserve binary compat:

```suggestion
        public ClrValueType GetStructValue(int index) => GetStructValue(index, false);

        public ClrValueType GetStructValue(int index, bool interior)
```

Comment (leculver) [reply src/Microsoft.Diagnostics.Runtime/src/Common/ClrArray.cs:202]: @chrisnas This change LGTM, if you wouldn't mind committing the above change.

Thanks!

Comment (chrisnas) [reply src/Microsoft.Diagnostics.Runtime/src/Common/ClrArray.cs:202]: I hesitated between the two solutions when I wrote it  :^)
I will do the change tomorrow

**Comment by leculver** [comment]
@chrisnas Shipped here:  https://www.nuget.org/packages/Microsoft.Diagnostics.Runtime/2.0.142501

**Pull request merged**